frontend/common/access/components/TeamAccess.tsx

```diff
@@ -38,7 +38,7 @@ export function TeamAccess(props: {
           label: t('Team name'),
         },
       }}
-      toolbarNameColumnFiltersValues={{ label: t('Team name'), query: 'team__name' }}
+      toolbarNameColumnFiltersValues={{ label: t('Team name'), query: 'team__name__icontains' }}
```

Comment:
@vidyanambiar, do we need to verify if the other APIs support icontains as well, or is it a feature that's intrinsic to all of them?

Reply:
Since this comes from DAB, it applies to all the services. I did a quick check on AWX/EDA resources as well and it seems to work as expected. 👍